calculate_means_wrangler.py

```diff
@@ -35,29 +33,25 @@ def lambda_handler(event, context):
         # Set up clients
         sqs = boto3.client("sqs", region_name="eu-west-2")
         lambda_client = boto3.client("lambda", region_name="eu-west-2")
-        sns = boto3.client("sns", region_name="eu-west-2")
 
         # ENV vars
         config, errors = InputSchema().load(os.environ)
         if errors:
             raise ValueError(f"Error validating environment params: {errors}")
-
-        logger.info("Vaildated params")
-
-        # Reads in Data from SQS Queue
-        response = get_sqs_message(config['queue_url'])
-        if "Messages" not in response:
-            raise NoDataInQueueError("No messages in queue")
-
-        message = response["Messages"][0]
-        message_json = json.loads(message["Body"])
-        receipt_handle = message["ReceiptHandle"]
-
-        logger.info("Succesfully retrieved data from sqs")
-
-        data = pd.DataFrame(message_json)
-
-        logger.info("Input data converted to dataframe")
+        in_file_name = config["in_file_name"]
+        out_file_name = config["out_file_name"]
+        incoming_message_group = config['incoming_message_group']
+        logger.info("Validated params")
```

Comment:
Possibly move the logger from inside the config block, just for readability if nothing else.